alert.html

```diff
@@ -0,0 +1,44 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../vui-colors/colors.html">
+
+<dom-module id="vui-alert">
+    <template>
+        <style>
+            :root {
+                --very-red: #ff7962;
+                --less-red: #ffeeec;
+                --border-width: 5px;
+                --border-radius: 3px;
+                --padding: 6px;
+            }
+            
+            .container {
+                background-color: var(--less-red);
+                border-width: 0;
+                border-left: var(--border-width) solid var(--very-red);
+                border-bottom-right-radius: var(--border-radius);
+                border-top-right-radius: var(--border-radius);
+                padding: var(--padding);
+            }
+            
+            .container [dir='rtl'] {
+		        border-left-width: 0;
+		        border-right: var(--border-width) solid var(--very-red);
+		        border-bottom-right-radius: 0;
+		        border-top-right-radius: 0;
+		        border-bottom-left-radius: var(--border-radius);
+		        border-top-left-radius: var(--border-radius);
+	        }
+        </style>
+        
+        <div class="container">
```

Comment:
...this element is at risk of being styled unexpected by some other style on the page because `.container` is pretty generic name.